addons/interaction/CfgVehicles.hpp

```diff
@@ -68,6 +68,14 @@ class CfgVehicles {
                     };
                 };
 
+                class ACE_TakeLauncher {
+                    displayName = CSTRING(takeLauncher);
+                    condition = QUOTE([ARR_3(_player,_target,(secondaryWeapon _target) param [ARR_2(0,'')])] call FUNC(canTakeLauncher));
+                    statement = QUOTE([ARR_3(_player,_target,(secondaryWeapon _target) param [ARR_2(0,'')])] call FUNC(takeLauncher));
```

Comment:
A unit only ever has one launcher equipped, why pass the launcher as a parameter to the function?
`secondaryWeapon` also already returns `""` when no launcher is equipped, why use `param`?